src/ctree-app/ctree-landing.html

```diff
@@ -533,33 +594,59 @@
           <p>W</p>
         </div>
         <div id="features" class="section-styling">
-          <div class= "center-div">
-            <h1 align="center">Features</h1>
-            <hr id="features-horizontal-line"/>
-            <!-- grid layout to be added here with icons and descriptions. -->
-          </div>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-        </div>
+          <h1 align="center">Features</h1>         
+          <!-- grid layout to be added here with icons and descriptions. -->
+          <div class="features-content">
+            <div class="collaboration-search-col">
+              <div class="collaboration-section">
```

Comment:
The documentation for iron-list stated that 

> iron-list should be used when a page has significantly more DOM nodes than the ones visible on the screen. 

under the **When should <iron-list> be used?** section. So, I went with dom-repeat since there are only four groups repeating in the features section. The items are still in an array, see PR #132 as a reference.

Reply:
Sure, dom-repeat makes sense, although I thought you'd need to do more work to get things to flow correctly, but maybe I'm wrong.